Gruntfile.js

```diff
@@ -360,7 +360,7 @@ module.exports = function(grunt) {
   /**
    * Create custom task aliases and combinations.
    */
-  grunt.registerTask('compile-cf', ['bower:cf', 'concat:cf-less', 'concat:topdocIcons']);
+  grunt.registerTask('compile-cf', ['bower:cf', 'concat:topdocIcons']);
```

Comment:
Hmmm, maybe we should rename the `compile-cf` task because we're not actually compiling it any more.